src/bin/ci/testConformanceLocal.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -e
+# import in shellcheck / CI / IntelliJ compatible ways
+# shellcheck source=/dev/null
+source "${BASH_SOURCE%/*}/test.inc.sh" || source test.inc.sh
+
+cromwell::build::setup_everyone_environment
+
+cromwell::build::setup_conformance_environment
+
+cromwell::build::assemble_jars
+
+CENTAUR_CWL_RUNNER_MODE="local"
+
+# Export variables used in conf files and commands
+export CENTAUR_CWL_RUNNER_MODE
+
+shutdown_cromwell() {
+    if [ -n "${CROMWELL_PID+set}" ]; then
+        cromwell::build::kill_children "${CROMWELL_PID}"
```

Comment:
not sure about the name 😬

Reply:
`kill_cromwell_server` maybe...

Reply:
The command kills a unix process ID and any of the child UNIX process IDs, especially those created from subshells. It's used for killing `funnel` and its `nohup`, `tail`, `cromwell`, etc.

Reply:
Going to try out `kill_tree`.